harmony-api/src/services/postService.ts

```diff
@@ -88,6 +111,9 @@ export const validateBody = (postData: Post): Array<string> => {
   if (!postData.body) {
     return ['body field is required to create a new discussion or review post']
   }
+  if (postData.title.length > 2000) {
+    return ['title field must be less than 1000 characters']
```

Comment:
This should be for the body right?

```suggestion
  if (postData.body.length > 2000) {
    return ['body field must be less than 1000 characters']
```